src/features/soft_navigations/instrument/index.js

```diff
@@ -23,7 +23,7 @@ export class Instrument extends InstrumentBase {
     const historyEE = wrapHistory(this.ee)
     const eventsEE = wrapEvents(this.ee)
 
-    const trackURLChange = () => handle('newURL', [now(), '' + window.location], undefined, this.featureName, this.ee)
+    const trackURLChange = () => handle('newURL', [TimeKeeper.now(), '' + window.location], undefined, this.featureName, this.ee)
```

Comment:
Can you actually change this **and** 'newDom' to use performance.now()? Accurate is important to interaction and tying events in. I think there's a larger cleanup audit needed to ensure JSErrors and Ajax are tracked more precisely as well when they pipe into SN for timeline evaluation, but for now this'll keep the precision of interactions to `DOMHighResTimestamp`.
(UI event which _starts_ interactions, and `trackURLChangeEvent` handler for popstate also uses the `timeStamp` property of events which are `DOMHighResTimestamp` -- this is what we want.)

Reply:
Can you open a separate ticket for that work? I don't think it needs to be included here and I don't want to add more changes than necessary.